types/react-custom-scrollbars/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for react-css-transition-replace 2.0.1
+// Type definitions for react-custom-scrollbars 4.1.1
```

Comment:
since this is a major upgrade, please create a new folder for v2, and keep a copy of the old files in it. see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version